diff --git a/chrome/services/qrcode_generator/qrcode_generator_service_impl.cc b/chrome/services/qrcode_generator/qrcode_generator_service_impl.cc
index 1094d0e20ed97cc62b41782426dedaeced8f3fd1..7533098f956849e04c68401da8c4121aadb1f987 100644
--- a/chrome/services/qrcode_generator/qrcode_generator_service_impl.cc
+++ b/chrome/services/qrcode_generator/qrcode_generator_service_impl.cc
@@ -79,6 +79,7 @@ void QRCodeGeneratorServiceImpl::DrawDino(SkCanvas* canvas,
                                           const SkPaint& paint_foreground,
                                           const SkPaint& paint_background) {
   int dino_width_px = pixels_per_dino_tile * dino_image::kDinoHeight;
+  BRAVE_QRCODE_GENERATOR_SERVICE_IMPL_DRAW_DINO
   int dino_height_px = pixels_per_dino_tile * dino_image::kDinoHeight;
 
   // If we request too big a dino, we'll clip. In practice the dino size
